chaincfg/params.go

```diff
@@ -297,6 +299,7 @@ var MainNetParams = Params{
 	GenesisHash:              &genesisHash,
 	PowLimit:                 mainPowLimit,
 	PowLimitBits:             0x1d00ffff,
+	EnforceBIP94:             false,
```

Comment:
IMO we don't need to add `false` to the existing params as that's the default value for a boolean.